lib/ex_unit/lib/ex_unit/assertions.ex

```diff
@@ -353,7 +353,8 @@ defmodule ExUnit.Assertions do
 
   defp pinned_vars([]), do: ""
   defp pinned_vars(pins) do
-    content = Enum.map(pins, fn(var) ->
+    content = Enum.uniq_by(pins, &Macro.to_string/1)
```

Comment:
I had originally had that basically, but I figured this was safer and more explicit. Should I change it still?